packages/node-red-s1seven-api/README.md

```diff
@@ -12,6 +12,470 @@ To notarize, hash or validate a certificate, simply pass in a valid JSON certifi
 
 Each node simply takes the required input via the config ui or the `msg` object, and outputs the `data` property of the response.
 
+## Authentication
+
+To authenticate a request, the api nodes look for an access token in the `msg.accessToken` property or in the global context `s1sevenAccessToken` property. The access token is automatically set in the global context when a request is made from the `get access token` node. To use the `get access token` node, simply add your application `clientId` and `clientSecret` to the config node, and send a request. To learn more, click [here](https://manual.s1seven.com/automation/#applications-and-access-tokens).
+
+Access tokens expire after 24 hours. There are several ways to automate the renewal of access tokens, one example can be seen below:
+
+![example](./images/example_auth.png)
+
+In the screenshot, each api node has 2 outputs, the first labeled `success`, and the second `fail`. The `fail` output is linked to a `switch` node. The `switch` looks at the property `msg.payload.statusCode` and checks to see if the code is `401`, which means the request is unauthorized. If the code is `401`, it sends the payload to the `get access token` node, which automatically request a new access token and makes available in the global context, along with the `mode`.
+
+![example](./images/switch.png)
+
+The above workflow can be imported using the following JSON file:
+
+```json
+[
+    {
+        "id": "4e8475bd9d88ccaf",
+        "type": "inject",
+        "z": "155da1287ae6504d",
+        "name": "",
+        "props": [
+            {
```

Comment:
For the sake of simplicity you could make this example cleaner by removing all nodes not useful for the authentication example.
Only 1 node that accesses the API with required authentication (apart from get access token) is needed don't you think ?

Reply:
Good idea 👍🏼